src/battle_script_commands.c

```diff
@@ -14998,8 +14998,13 @@ static void Cmd_handleballthrow(void)
             }
             else
             {
+                #if P_CATCH_CURVE >= GEN_6
+                odds = 16581375 / (odds * odds * odds);
+                odds = 65536 / Sqrt(Sqrt(Sqrt(Sqrt(odds))));
+                #else
```

Comment:
Can you change it to a normal `if`? We are trying to minimize preprocs

Reply:
Ok